jest_config/jest.config.e2e.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why have a separate config for E2E tests? Can we not combine this into one config?

Reply:
Maybe? I just couldn't figure out how to run them on separate commands without